tests/shared/s3-packages.spec.js

```diff
@@ -0,0 +1,988 @@
+/*
+ * Copyright 2016 resin.io
```

Comment:
What is the convention here? I thought the Copyright date in the header actually reflects the year that the file was first created?